clouddriver-web/src/test/groovy/com/netflix/spinnaker/clouddriver/MainSpec.java

```diff
@@ -24,7 +24,12 @@
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @SpringBootTest(classes = {Main.class})
-@TestPropertySource(properties = {"redis.enabled = false", "sql.enabled = false"})
+@TestPropertySource(
+    properties = {
+      "redis.enabled = false",
+      "sql.enabled = false",
+      "spring.cloud.config.server.git.uri = test"
```

Comment:
Is this needed to have the test pass?  In general, `MainSpec` tries to specify the minimum config needed for a successful startup, the main goal being to catch when people accidentally make a config parameter required (ex: by gating a bean behind a parameter, but autowiring it into a core class).  For that reason, I'd rather omit that config parameter here.

Reply:
I moved where the new `ConfigFileService` configuration was getting done so its conditional on a config source being defined. That cleaned this up and gets all the Config Server related config in one place.